flutter/shell/platform/tizen/flutter_tizen_engine.h

```diff
@@ -48,6 +48,7 @@ namespace flutter {
 constexpr FlutterViewId kImplicitViewId = 0;
 
 class FlutterTizenView;
+class FlutterTizenDisplayMonitor;
```

Comment:
I think it is acceptable in this case though but we usually use avoiding forward declarations if possible. How about including headers?
https://google.github.io/styleguide/cppguide.html#Forward_Declarations